### ISSUE TITLE
загрузка графиков

### ISSUE THREAD

**Issue by vomma-opt** [SBProX/SBProX-Dev#1482]
Добрый день, а нельзя сделать так чтобы при каждом запуске программа не загружала графики по 5 минут? Можно сделать так чтобы открыл программу и загружались только недостающие свечки, долго приходится ждать пока все загрузится, и так при каждом запуске программы.

**Comment by vomma-opt** [comment]
и так каждый раз при запуске программы http://prntscr.com/tvi2us

**Comment by ikki1561** [comment]
поддерживаю

**Comment by ikki1561** [comment]
> Добрый день, а нельзя сделать так чтобы при каждом запуске программа не загружала графики по 5 минут? Можно сделать так чтобы открыл программу и загружались только недостающие свечки, долго приходится ждать пока все загрузится, и так при каждом запуске программы.

и выбор коннектора вынести на первое окно, не думаю что кто то постоянно его меняет, а то раза три нужно нажать продолжить чтобы пограма запустилась

**Comment by vomma-opt** [comment]
и нагрузка на Ваши сервера уменьшится, клиенты не будут постоянно подтягивать данные

**Comment by smart3353** [comment]
лайк

**Comment by vomma-opt** [comment]
> > Добрый день, а нельзя сделать так чтобы при каждом запуске программа не загружала графики по 5 минут? Можно сделать так чтобы открыл программу и загружались только недостающие свечки, долго приходится ждать пока все загрузится, и так при каждом запуске программы.
> 
> и выбор коннектора вынести на первое окно, не думаю что кто то постоянно его меняет, а то раза три нужно нажать продолжить чтобы програма запустилась

да, тут тоже согласен перед тем как увидеть графики, приходится 4 окна закрыть с коннектором, с загрузкой процессора и ОЗУ с вопросом все ли вкладки одновременно загрузить или по отдельности. Можно эти параметры перенести во вкладку Общее - один раз под себя настроил и больше к этим настройкам не возвращаешься, каждый запуск одни и те же всплывающие окна, пожалуйста прислушайтесь. Пользователю нужно предоставить выбор и возможность настройки

**Comment by serb81** [comment]
Поддерживаю
